test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -69,5 +69,14 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
 	$dirEntry.Length | Should BeGreaterThan 0
     }
 
+	It "Should give an error when Verb parameter is used" {
+	$err = $null
+	try {
+		Start-Process -Verb runas -FilePath $pingCommand -ArgumentList $pingParam
```

Comment:
Indentation is messed up, and this does not follow the pattern @JamesWTruher has specified for tests that throw, please follow his guidelines.